src/fsharp/FSharp.Build/Microsoft.FSharp.NetSdk.props

```diff
@@ -40,6 +40,7 @@ WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and
     <TreatWarningsAsErrors Condition="'$(TreatWarningsAsErrors)' == '' ">false</TreatWarningsAsErrors>
     <WarningLevel Condition=" '$(WarningLevel)' == '' ">3</WarningLevel>
     <WarningsAsErrors>3239;$(WarningsAsErrors)</WarningsAsErrors>
+    <WarnOn>$(WarnOn);3390</WarnOn>
```

Comment:
I think this is already handled in the compiler

Reply:
@cartermp , in the compiler it is default off, this turns it on for projects built with updated tooling.

Reply:
@cartermp , I think this value should really be set in the templates, doing what I am doing here is sort of ... counterproductive.